### DISCUSSION TITLE
Switching package between stable/dev

### DISCUSSION THREAD

**Discussion by bilogic** [composer/composer#11355] [Q&A / Support]
```JSON
"require": {
    "bilogic/package1": "^0.0",
},
"repositories": [
    {
        "type": "path",
        "url": "path/to/repo1/for/dev/packages/*/*",
        "canonical": false
    },
    {
        "type": "composer",
        "url": "https:\/\/repo2.com"
    }
]
```

1. I have the `bilogic/package1` on both `repo1` and `repo2`
2. Is there a way to load from `repo1` instead of `repo2` at the flick of a switch without needing to modify the package's version?

**Comment by Seldaek** [comment]
Nope not really, look into using https://github.com/franzliedke/studio maybe?
